site/docs/red-team/owasp-llm-top-10.md

```diff
@@ -235,20 +345,27 @@ redteam:
     - owasp:llm:09
 ```
 
-## 10. Model Theft (LLM10)
+## 10. Unbounded Consumption (LLM10)
```

Comment:
if you'd like, you can add the `divergent-repetition` plugin here.  I'll merge it asap